src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -1075,7 +1081,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
       return false
     }
 
-    if (isRequirementMet && !isSkin && isSmartWearable && modelSize && modelSize > MAX_SMART_WEARABLE_FILE_SIZE) {
+    if (isRequirementMet && !isSkin && isSmartWearable && finalSize && !isSmartWearableFileSizeValid(finalSize)) {
```

Comment:
can it be a smart wearable and skin?

Reply:
mmmm, it's a good point. the smart wearables can be skin but have the limitation for the `js` files of 3MB max. 

will update the condition to have a better validation